frontend/src/components/homepage/components/popular-users/styles.module.scss

```diff
@@ -1,32 +1,40 @@
 .root {
   min-height: 290px;
   margin-bottom: 40px;
+}
+
+.title {
+  font-size: 24px;
+  margin: 0 0 25px;
+}
 
-  .title {
-    font-size: 24px;
-    margin: 0 0 25px;
+.groupPodcastes {
+  justify-content: space-between;
+
+  li {
+    margin-right: 0px;
   }
+}
 
-  .usersRow {
-    display: flex;
-    justify-content: space-between;
-    margin: 0;
-    padding: 0;
-    grid-gap: 35px;
-    overflow-x: auto;
+.usersRow {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  grid-gap: 35px;
+  overflow-x: auto;
 
-    li {
-      margin-bottom: 15px;
-    }
+  li {
+    margin-bottom: 15px;
+    margin-right: 35px;
```

Comment:
You have used `grid-gap` already. Why you started to use` margin-right`?
![image](https://user-images.githubusercontent.com/60543500/130826952-24b4cff0-9e77-483c-848d-4c2571fa7f61.png)